README.md

```diff
@@ -1,128 +1,131 @@
-
-Contributions, issues, and feature requests are welcome!
-Feel free to check the [issues page].
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-## ⭐️ Show your support <a name="support"></a>
-
-If you like this project you can contact me via my contact address and rate it.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-I would like to thank all the microverse team and my coding partners.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-## 📝 License <a name="license"></a>
-
-This project is [<a href = "MIT.md">MIT</a>](./MIT.md) licensed.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Today's To Do List</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Awesome Books\] ](#-awesome-books-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [Today's To Do List] <a name="about-project"></a>
+
+A Todo List app.
+
+Our application is a Todo List app that allows users to create, view, update, and delete tasks. Users can add tasks by entering a description and clicking the "Add" button. The task list is displayed, showing each task with a checkbox and options to edit or delete.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+1. Linters
+2. Webpack
+3. HTML5
+4. JS (ES6 synthax)
+5. CSS3
+
+### Key Features <a name="key-features"></a>
+
+1. Mark tasks as complete: Add functionality that allows users to mark tasks as completed. This can be done by adding a checkbox next to each task and updating its status accordingly.
+
+2. Delete Tasks: Allow users to delete tasks from the list after they are completed or no longer relevant. You can add a trash can icon next to each task to allow deletion.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+install node and vscode
+linters configuration
+webpack configuration
+
+### setup
+
+<p>Clone this repository to your desired folder</p>
+<a href ="git@github.com:GhostEsso/toDoList.git"></a>
+
+### Install
+
+Install this project with: -Git clone on your cd Desktop using my repository link above
+
+### Usage
+
+-To run the project, copy its path and run it on your online server.

```

Comment:
- [ ] I would suggest you please give instruction how user can clone the project, install it and run the project. For example:
` git clone git@github.com:GhostEsso/toDoList.git`
`npm install` to install all the dependencies
`npm start` to start the project on the browser
This is the way they clone, install and start the project as if new user visit you README who don't know programming then they can also easily run the project.